en/docs/fragments/saml-app-artifact-binding-config.md

```diff
@@ -0,0 +1,25 @@
+## SAML Configurations 
+
+Make the following changes to the created service provider.
```

Comment:
I don't think we need it since it is right after the registering step and the user will already be there in that screen.